content/tracing/setup/go.md

```diff
@@ -80,53 +100,60 @@ ___
 | SQL                 | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/database/sql | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/database/sql                      |
 | SQLx                | https://github.com/jmoiron/sqlx                                       | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/jmoiron/sqlx                      |
 
-### OpenTracing API
+___
+
+Make sure to take a look at our integrations package [godoc page][contrib godoc] for an in-depth look.
 
-Datadog APM client that implements an [OpenTracing][3] Tracer.
+## OpenTracing Support
```

Comment:
Perhaps we should move this section up so it's right after Manual Instrumentation?

Reply:
I did that, but personally I feel that Opentracing shouldn't be mixed together in-between the sections that address Datadog-specific instrumentation, since it's a separate topic.